jobs/candlepinQpidTestsJob.groovy

```diff
@@ -0,0 +1,35 @@
+import jobLib.rhsmLib
+
+String baseFolder = rhsmLib.candlepinJobFolder
+
+String githubOrg = binding.variables['CANDLEPIN_JENKINS_GITHUB_ORG'] ?: 'candlepin'
+
+def rhsmJob = job("$baseFolder/candlepin-pullrequest-spectests-qpid") {
+    previousNames('candlepin-pullrequest-spectests-qpid')
+    environmentVariables(CANDLEPIN_DATABASE: 'postgresql', CP_TEST_ARGS: '-q -r qpid_spec')
+    description("Compiles and deploys Candlepin and runs rspec tests against PostgreSQL with Qpid. Does not produce any installable products.")
+    label('docker')
+    concurrentBuild()
+    wrappers {
+        timeout {
+            noActivity(3600)
```

Comment:
If this is in seconds (and it looks to be; 3600 is 1 hour), this is probably cutting it a bit close. There are days where a network slowdown or hiccup will cause our spec tests to run long. Since running these takes so bloody long to begin with, I'd vote we increase this to 90-120 minutes to give us a bit more headroom in this regard.